libevmasm/Assembly.cpp

```diff
@@ -1427,6 +1430,14 @@ LinkerObject const& Assembly::assembleEOF() const
 				ret.linkReferences.insert(linkRef);
 				break;
 			}
+			case RelativeJump:
+			case ConditionalRelativeJump:
+			{
+				ret.bytecode.push_back(static_cast<uint8_t>(item.instruction()));
+				tagRef[ret.bytecode.size()] = item.realativeJumpOffset();
```

Comment:
Oh, I just realized that my suggested name was wrong: it's the tag ID, not the offset.

```suggestion
				tagRef[ret.bytecode.size()] = item.realativeJumpTagID();
```

But that just shows that this really needed to be named.